CONTRIBUTING.md

```diff
@@ -464,7 +473,7 @@ Frontend assets (TypeScript, JavaScript, CSS, and images) must be compiled in or
 
 ##### nvm and node
 
-First, be sure you are using recent versions of NodeJS and npm. We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your node environment:
+First, be sure you are using recent versions of node.js and npm. We recommend using [nvm](https://github.com/nvm-sh/nvm) to manage your node environment:
```

Comment:
I did some quick googling, and it appears the commonly used name is Node.js with a capital N: https://en.wikipedia.org/wiki/Node.js